Makefile.PL

```diff
@@ -26,11 +26,13 @@ requires        ('List::Util');
 requires        ('Encode');
 requires        ('File::Find');
 requires        ('IO::Socket::INET');
+requires        ('File::Copy::Recursive');
```

Comment:
I'm not feeling comfortable with adding a new dependency for a feature I've never wanted personally :) Maybe we can just use `cp -r SRC DEST` from shell?